docs/funding.md

```diff
@@ -0,0 +1,56 @@
+[< Back to Navigation](../README.md)
+
+# Funding
+
+## Funding eligibility during registration
+
+TODO: to be completed by NPQ reg team
+
+## Accepting an application
+
+When accepting an application we evaluate the `funded_place` attribute passed in the payload against the `funding_cap` of the cohort and `eligible_for_funding` status of the application:
+
+- The `funded_place` attribute in the accept application payload will be evaluated only for cohorts that have a `funding_cap`. 
+- An application must be `eligible_for_funding` if the application is being accepted for a `funded_place`.
```

Comment:
can we elaborate further here with examples, maybe:
- we can set `funded_place` true/false on `eligible_for_funding` true
- and `funded_place` false on `eligible_for_funding` false only